src/wallet/rpcwallet.cpp

```diff
@@ -3157,6 +3206,25 @@ static UniValue fundrawtransaction(const JSONRPCRequest& request)
                         "This boolean should reflect whether the transaction has inputs\n"
                         "(e.g. fully valid, or on-chain transactions), if known by the caller."
                     },
+                    {"solving_data", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED_NAMED_ARG, "Keys and scripts needed for producing a final transaction with a dummy signature. Used for fee estimation during coin selection.\n",
```

Comment:
This seems like an advanced feature, so maybe move to "options"?

Reply:
I don't think this really belongs as part of options. It's not like a setting, just additional data needed.